contrib/seeds/README.md

```diff
@@ -8,7 +8,7 @@ and remove old versions as necessary.
 
 The seeds compiled into the release are created from sipa's DNS seed data, like this:
 
-    curl -s http://bitcoin.sipa.be/seeds.txt > seeds_main.txt
+    curl -s http://bitcoin.sipa.be/seeds.txt.gz | gunzip > seeds_main.txt
```

Comment:
Found this thead: https://lists.gnu.org/archive/html/bug-automake/2009-10/msg00012.html